apple/internal/processor.bzl

```diff
@@ -242,8 +242,8 @@ def _bundle_partial_outputs_files(
             if (invalid_top_level_dirs and
                 not _is_parent_dir_valid(invalid_top_level_dirs, parent_dir)):
                 fail(("Error: For %s bundles, the following top level " +
-                      "directories are invalid: %s") %
-                     (platform_type, ", ".join(invalid_top_level_dirs)))
+                      "directories are invalid: %s, check input files: %s") %
+                     (platform_type, ", ".join(invalid_top_level_dirs), files))
```

Comment:
can you add this above `fail`?

```
file_paths = "\n".join([f.path for f in files.to_list()])
```

and then use `file_paths` instead of `files` (remember to put a `\n` after `check intput files:` so the paths start from a new line)